packages/flutter/lib/src/painting/matrix_utils.dart

```diff
@@ -281,6 +288,20 @@ class TransformProperty extends DiagnosticsProperty<Matrix4> {
 
   @override
   String valueToString({ TextTreeConfiguration parentConfiguration }) {
+    if (value != null) {
```

Comment:
why do you check value != null here? if it equals null, it'll skip this block, do the next block, and immediately crash since we do an unguarded dereference below.

I would just assert != null, unless we are supposed to handle null here.